### PR TITLE
ignore local .log files in git commit

### PR DESCRIPTION
**Pull request by criticalAY**
## Purpose / Description
Dont allow the local log files to be added to the git commit so adding them to git ignore because while committing the log files were getting added to the git commit and these are the local files which get generated when we have our local system issue like out of memory so there is no point in allowing them to be added to the commit 

## Fixes
Fixes _Link to the issues._

## How Has This Been Tested?
would ignore log files

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by lukstbit** [comment]
I don't remember having ever seeing .log files in the project and I don't want to add something to the .gitignore file unless is clearly an issue. Did you have specific issues with this?

**Comment by criticalAY** [comment]
> I don't remember ever seeing .log files in the project and I don't want to add something to the .gitignore file unless is clearly an issue. Did you have specific issues with this?

Yes, many times when the android studio has memory issues (most of the cases when I left the system on for too long and android studio running in the background) I usually discarded the changes when and went ahead with the commit but I think it is better added to ignore file

**Comment by BrayanDSO** [comment]
Never saw these. Have you tried adding it to your `exclude` file? https://git-scm.com/docs/gitignore#_description

**Comment by criticalAY** [comment]
I will wait for these to pop up again to examine these

**Comment by BrayanDSO** [comment]
Okay, so I'm gonna draft this

**Comment by criticalAY** [comment]
@BrayanDSO @lukstbit 
```
+#
# There is insufficient memory for the Java Runtime Environment to continue.
# Native memory allocation (mmap) failed to map 637534208 bytes for G1 virtual space
# Possible reasons:
#   The system is out of physical RAM or swap space
#   The process is running with CompressedOops enabled, and the Java Heap may be blocking the growth of the native heap
# Possible solutions:
#   Reduce memory load on the system
#   Increase physical memory or swap space
#   Check if swap backing store is full
#   Decrease Java heap size (-Xmx/-Xms)
#   Decrease number of Java threads
#   Decrease Java thread stack sizes (-Xss)
#   Set larger code cache with -XX:ReservedCodeCacheSize=
#   JVM is running with Zero Based Compressed Oops mode in which the Java heap is
#     placed in the first 32GB address space. The Java Heap base address is the
#     maximum limit for the native heap growth. Please use -XX:HeapBaseMinAddress
#     to set the Java Heap base and to place the Java Heap above 32GB virtual address.
# This output file may be truncated or incomplete.
#
#  Out of Memory Error (os_windows.cpp:3769), pid=45096, tid=44364
#
# JRE version: OpenJDK Runtime Environment (17.0.6) (build 17.0.6+0-b2043.56-9586694)
# Java VM: OpenJDK 64-Bit Server VM (17.0.6+0-b2043.56-9586694, mixed mode, tiered, compressed oops, compressed class ptrs, g1 gc, windows-amd64)
# No core dump will be written. Minidumps are not enabled by default on client versions of Windows
#

---------------  S U M M A R Y ------------

Command Line: --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.lang.invoke=ALL-UNNAMED --add-opens=java.prefs/java.util.prefs=ALL-UNNAMED --add-opens=java.base/java.nio.charset=ALL-UNNAMED --add-opens=java.base/java.net=ALL-UNNAMED --add-opens=java.base/java.util.concurrent.atomic=ALL-UNNAMED -Xmx3072M -Dfile.encoding=UTF-8 -Duser.country=IN -Duser.language=en -Duser.variant -javaagent:C:\Users\mailt\.gradle\wrapper\dists\gradle-8.1.1-bin\9wiye5v2saajue4irfo8ybqfp\gradle-8.1.1\lib\agents\gradle-instrumentation-agent-8.1.1.jar org.gradle.launcher.daemon.bootstrap.GradleDaemon 8.1.1

Host: AMD Ryzen 5 5600H with Radeon Graphics         , 12 cores, 7G,  Windows 11 , 64 bit Build 22621 (10.0.22621.1778)
Time: Sat Jul  8 00:10:46 2023 India Standard Time elapsed time: 124.755751 seconds (0d 0h 2m 4s)

---------------  T H R E A D  ---------------

Current thread (0x000002a7f86db3e0):  VMThread "VM Thread" [stack: 0x000000bd14500000,0x000000bd14600000] [id=44364]

Stack: [0x000000bd14500000,0x000000bd14600000]
Native frames: (J=compiled Java code, j=interpreted, Vv=VM code, C=native code)
V  [jvm.dll+0x683b4a]
V  [jvm.dll+0x842654]
V  [jvm.dll+0x843dfe]
V  [jvm.dll+0x844463]
V  [jvm.dll+0x249b75]
V  [jvm.dll+0x680a19]
V  [jvm.dll+0x674f0a]
V  [jvm.dll+0x309acb]
V  [jvm.dll+0x310f86]
V  [jvm.dll+0x3608ee]
V  [jvm.dll+0x360b1f]
V  [jvm.dll+0x2dfbf8]
V  [jvm.dll+0x2ddec4]
V  [jvm.dll+0x2dd4cc]
V  [jvm.dll+0x321f3b]
V  [jvm.dll+0x84893b]
V  [jvm.dll+0x849674]
V  [jvm.dll+0x849b8d]
V  [jvm.dll+0x849f64]
V  [jvm.dll+0x84a030]
V  [jvm.dll+0x7f25fa]
V  [jvm.dll+0x682985]
C  [ucrtbase.dll+0x29363]
C  [KERNEL32.DLL+0x126ad]
C  [ntdll.dll+0x5a9f8]

VM_Operation (0x000000bd1f7f7bb0): G1CollectForAllocation, mode: safepoint, requested by thread 0x000002a786845690


---------------  P R O C E S S  ---------------

Threads class SMR info:
_java_thread_list=0x000002a7fd3f32e0, length=228, elements={
0x000002a7d9f1b310, 0x000002a7f8745500, 0x000002a7f87460b0, 0x000002a7f8742c10,
0x000002a7f9037610, 0x000002a7f903e0e0, 0x000002a7f903ea90, 0x000002a7f90411b0,
0x000002a7f9045d70, 0x000002a7f9050790, 0x000002a7f913efc0, 0x000002a7f9292af0,
0x000002a7fc7a08a0, 0x000002a7fadc7030, 0x000002a7fc175cb0, 0x000002a7fb78d860,
0x000002a7fba67c90, 0x000002a7fadc6880, 0x000002a7fadc5a10, 0x000002a7fadc5070,
0x000002a7fadc5540, 0x000002a7fadc63b0, 0x000002a7fbcad8a0, 0x000002a7fbca9ee0,
0x000002a7fbcaff20, 0x000002a7fbcaf580, 0x000002a7fbcaf0b0, 0x000002a7fbcab6f0,
0x000002a7fbcaa3b0, 0x000002a7fbcae240, 0x000002a7fbca9070, 0x000002a7fbcafa50,
0x000002a7fbcacf00, 0x000002a7fbca9540, 0x000002a7fbcaad50, 0x000002a7fbca9a10,
0x000002a7fbcae710, 0x000002a7fbcaebe0, 0x000002a7fbcaa880, 0x000002a7fbcaca30,
0x000002a7fbcb03f0, 0x000002a7fbcb08c0, 0x000002a7fbcadd70, 0x000002a7fd2dff90,
0x000002a7fd2da8f0, 0x000002a7fd2de780, 0x000002a7fd2e0930, 0x000002a7fd2e0460,
0x000002a7fd2d90e0, 0x000002a7fd2df120, 0x000002a7fd2dcf70, 0x000002a7fd2df5f0,
0x000002a7fd2db290, 0x000002a7fd2dec50, 0x000002a7fd2d9a80, 0x000002a7fd2dd910,
0x000002a7fd2dfac0, 0x000002a7fd2ddde0, 0x000002a7fd2da420, 0x000002a7fd2dc100,
0x000002a7fd2dadc0, 0x000002a7fd2dcaa0, 0x000002a7fd2db760, 0x000002a7fd2dbc30,
0x000002a7fd2dc5d0, 0x000002a7fd2de2b0, 0x000002a7fba91790, 0x000002a7fba947b0,
0x000002a7fba91c60, 0x000002a7fba93940, 0x000002a7fba94c80, 0x000002a7fba92130,
0x000002a7fba92ad0, 0x000002a7fba912c0, 0x000002a7fba92600, 0x000002a7fba92fa0,
0x000002a7fba93470, 0x000002a7fba93e10, 0x000002a7fba942e0, 0x000002a781a87e40,
0x000002a781a88310, 0x000002a781a84950, 0x000002a781a81e00, 0x000002a781a86b00,
0x000002a781a84e20, 0x000002a781a84480, 0x000002a781a85c90, 0x000002a78221a8c0,
0x000002a78221b260, 0x000002a78221a3f0, 0x000002a7818bec60, 0x000002a7818be790,
0x000002a7fb83d3d0, 0x000002a7fb83c090, 0x000002a7fb83c560, 0x000002a7fb83cf00,
0x000002a7ff9e1110, 0x000002a7ff9e4130, 0x000002a7ff9e4600, 0x000002a7ff9e15e0,
0x000002a7ff9e3c60, 0x000002a7ff9e32c0, 0x000002a7ff9e3790, 0x000002a7ff9e2df0,
0x000002a7ff9e4ad0, 0x000002a7ff9e1ab0, 0x000002a7ff9e1f80, 0x000002a7ff9e2450,
0x000002a7ff9e2920, 0x000002a7fd6d8770, 0x000002a7fd6d7dd0, 0x000002a7febbcf90,
0x000002a7febbcac0, 0x000002a7febbd930, 0x000002a781a887e0, 0x000002a7fd6d8c40,
0x000002a7fd6d9110, 0x000002a7fd6d7900, 0x000002a786845b60, 0x000002a7868439b0,
0x000002a7febbc120, 0x000002a7febbc5f0, 0x000002a7feda77a0, 0x000002a7feda9480,
0x000002a786976fd0, 0x000002a7869774a0, 0x000002a786977970, 0x000002a7feda72d0,
0x000002a7feda9950, 0x000002a7feda9e20, 0x000002a7feda8fb0, 0x000002a7fedaa2f0,
0x000002a786844cf0, 0x000002a786846030, 0x000002a7868451c0, 0x000002a7fd3c9a20,
0x000002a785e2c490, 0x000002a7fedaa7c0, 0x000002a7faa1db10, 0x000002a785dcfed0,
0x000002a785dd03a0, 0x000002a786977e40, 0x000002a783d180d0, 0x000002a7feda7c70,
0x000002a7869752f0, 0x000002a786974950, 0x000002a786976630, 0x000002a786974e20,
0x000002a786975c90, 0x000002a7869757c0, 0x000002a786978310, 0x000002a786976160,
0x000002a7f85bb230, 0x000002a7f85bad60, 0x000002a7f85bc570, 0x000002a7f85ba890,
0x000002a786844350, 0x000002a786844820, 0x000002a786845690, 0x000002a786846500,
0x000002a786842b40, 0x000002a786843010, 0x000002a7868434e0, 0x000002a7ff6993a0,
0x000002a7ff69a6e0, 0x000002a7ff6dba90, 0x000002a7ff6dbf60, 0x000002a7ff6db0f0,
0x000002a7ff6dc430, 0x000002a7818bf130, 0x000002a785dcfa00, 0x000002a7feda8140,
0x000002a7f85bb700, 0x000002a7f85bc0a0, 0x000002a7f85bca40, 0x000002a7f85b9080,
0x000002a7f85b9550, 0x000002a7f85b9a20, 0x000002a7f85b9ef0, 0x000002a7f85ba3c0,
0x000002a7fc552c30, 0x000002a7fc54f740, 0x000002a7fc550a80, 0x000002a781a83140,
0x000002a7feda8610, 0x000002a7ff6dc900, 0x000002a7ff6db5c0, 0x000002a7feda8ae0,
0x000002a783d18a70, 0x000002a783d18f40, 0x000002a783d19410, 0x000002a785e2baf0,
0x000002a785e2c960, 0x000002a785e2ce30, 0x000002a785e2b620, 0x000002a785e2bfc0,
0x000002a7fd08d4a0, 0x000002a7fd08cfd0, 0x000002a7fb5540a0, 0x000002a7fb554f10,
0x000002a7fb554a40, 0x000002a7fd08ecb0, 0x000002a7818bd920, 0x000002a7818bddf0,
0x000002a7818be2c0, 0x000002a78534cce0, 0x000002a78534b000, 0x000002a78534d1b0,
0x000002a781ee50b0, 0x000002a781a83610, 0x000002a781a83ae0, 0x000002a7fd6a2ef0,
0x000002a7fd6a5570, 0x000002a78534d680, 0x000002a78534b9a0, 0x000002a7fc551dc0,
0x000002a7fc54fc10, 0x000002a786685450, 0x000002a786682dd0, 0x000002a78b940330,
0x000002a78b93eb20, 0x000002a786684110, 0x000002a7866832a0, 0x000002a78b93dcb0,
0x000002a78b93e180, 0x000002a78b940cd0, 0x000002a7fd08f180, 0x000002a7fd6fbd90
}

Java Threads: ( => current thread )
  0x000002a7d9f1b310 JavaThread "main" [_thread_blocked, id=41472, stack(0x000000bd13f00000,0x000000bd14000000)]
  0x000002a7f8745500 JavaThread "Reference Handler" daemon [_thread_blocked, id=44740, stack(0x000000bd14600000,0x000000bd14700000)]
  0x000002a7f87460b0 JavaThread "Finalizer" daemon [_thread_blocked, id=50340, stack(0x000000bd14700000,0x000000bd14800000)]
  0x000002a7f8742c10 JavaThread "Signal Dispatcher" daemon [_thread_blocked, id=45696, stack(0x000000bd14800000,0x000000bd14900000)]
  0x000002a7f9037610 JavaThread "Attach Listener" daemon [_thread_blocked, id=38044, stack(0x000000bd14900000,0x000000bd14a00000)]
  0x000002a7f903e0e0 JavaThread "Service Thread" daemon [_thread_blocked, id=3944, stack(0x000000bd14a00000,0x000000bd14b00000)]
  0x000002a7f903ea90 JavaThread "Monitor Deflation Thread" daemon [_thread_blocked, id=48608, stack(0x000000bd14b00000,0x000000bd14c00000)]
  0x000002a7f90411b0 JavaThread "C2 CompilerThread0" daemon [_thread_blocked, id=38116, stack(0x000000bd14c00000,0x000000bd14d00000)]
  0x000002a7f9045d70 JavaThread "C1 CompilerThread0" daemon [_thread_blocked, id=44412, stack(0x000000bd14d00000,0x000000bd14e00000)]
  0x000002a7f9050790 JavaThread "Sweeper thread" daemon [_thread_blocked, id=47072, stack(0x000000bd14e00000,0x000000bd14f00000)]
  0x000002a7f913efc0 JavaThread "Common-Cleaner" daemon [_thread_blocked, id=50988, stack(0x000000bd14f00000,0x000000bd15000000)]
  0x000002a7f9292af0 JavaThread "Notification Thread" daemon [_thread_blocked, id=49556, stack(0x000000bd15000000,0x000000bd15100000)]
  0x000002a7fc7a08a0 JavaThread "Daemon health stats" [_thread_blocked, id=47156, stack(0x000000bd15800000,0x000000bd15900000)]
  0x000002a7fadc7030 JavaThread "Incoming local TCP Connector on port 61754" [_thread_in_native, id=41684, stack(0x000000bd15200000,0x000000bd15300000)]
  0x000002a7fc175cb0 JavaThread "Daemon periodic checks" [_thread_blocked, id=52988, stack(0x000000bd15700000,0x000000bd15800000)]
  0x000002a7fb78d860 JavaThread "Daemon" [_thread_blocked, id=45448, stack(0x000000bd15900000,0x000000bd15a00000)]
  0x000002a7fba67c90 JavaThread "Handler for socket connection from /127.0.0.1:61754 to /127.0.0.1:61755" [_thread_in_native, id=40172, stack(0x000000bd15a00000,0x000000bd15b00000)]
  0x000002a7fadc6880 JavaThread "Cancel handler" [_thread_blocked, id=17412, stack(0x000000bd15b00000,0x000000bd15c00000)]
  0x000002a7fadc5a10 JavaThread "Daemon worker" [_thread_blocked, id=17716, stack(0x000000bd15c00000,0x000000bd15d00000)]
  0x000002a7fadc5070 JavaThread "Asynchronous log dispatcher for DefaultDaemonConnection: socket connection from /127.0.0.1:61754 to /127.0.0.1:61755" [_thread_blocked, id=8836, stack(0x000000bd15d00000,0x000000bd15e00000)]
  0x000002a7fadc5540 JavaThread "Daemon client event forwarder" [_thread_blocked, id=50512, stack(0x000000bd15f00000,0x000000bd16000000)]
  0x000002a7fadc63b0 JavaThread "Cache worker for journal cache (C:\Users\mailt\.gradle\caches\journal-1)" [_thread_blocked, id=50780, stack(0x000000bd16700000,0x000000bd16800000)]
  0x000002a7fbcad8a0 JavaThread "File lock request listener" [_thread_in_native, id=51240, stack(0x000000bd16800000,0x000000bd16900000)]
  0x000002a7fbca9ee0 JavaThread "Cache worker for file hash cache (C:\Users\mailt\.gradle\caches\8.1.1\fileHashes)" [_thread_blocked, id=11764, stack(0x000000bd16a00000,0x000000bd16b00000)]
  0x000002a7fbcaff20 JavaThread "File watcher server" daemon [_thread_blocked, id=10796, stack(0x000000bd16b00000,0x000000bd16c00000)]
  0x000002a7fbcaf580 JavaThread "File watcher consumer" daemon [_thread_blocked, id=26780, stack(0x000000bd16c00000,0x000000bd16d00000)]
  0x000002a7fbcaf0b0 JavaThread "Cache worker for checksums cache (D:\Anki-Android\.gradle\8.1.1\checksums)" [_thread_blocked, id=1724, stack(0x000000bd16d00000,0x000000bd16e00000)]
  0x000002a7fbcab6f0 JavaThread "Cache worker for cache directory md-rule (C:\Users\mailt\.gradle\caches\8.1.1\md-rule)" [_thread_blocked, id=49232, stack(0x000000bd16e00000,0x000000bd16f00000)]
  0x000002a7fbcaa3b0 JavaThread "Cache worker for file content cache (C:\Users\mailt\.gradle\caches\8.1.1\fileContent)" [_thread_blocked, id=8184, stack(0x000000bd16f00000,0x000000bd17000000)]
  0x000002a7fbcae240 JavaThread "Cache worker for file hash cache (D:\Anki-Android\.gradle\8.1.1\fileHashes)" [_thread_blocked, id=47728, stack(0x000000bd17000000,0x000000bd17100000)]
  0x000002a7fbca9070 JavaThread "Cache worker for cache directory md-supplier (C:\Users\mailt\.gradle\caches\8.1.1\md-supplier)" [_thread_blocked, id=38820, stack(0x000000bd17100000,0x000000bd17200000)]
  0x000002a7fbcafa50 JavaThread "Cache worker for execution history cache (C:\Users\mailt\.gradle\caches\8.1.1\executionHistory)" [_thread_blocked, id=47132, stack(0x000000bd17200000,0x000000bd17300000)]
  0x000002a7fbcacf00 JavaThread "jar transforms" [_thread_blocked, id=34108, stack(0x000000bd17300000,0x000000bd17400000)]
  0x000002a7fbca9540 JavaThread "jar transforms Thread 2" [_thread_blocked, id=37992, stack(0x000000bd17400000,0x000000bd17500000)]
  0x000002a7fbcaad50 JavaThread "jar transforms Thread 3" [_thread_blocked, id=45368, stack(0x000000bd17600000,0x000000bd17700000)]
  0x000002a7fbca9a10 JavaThread "jar transforms Thread 4" [_thread_blocked, id=51356, stack(0x000000bd17700000,0x000000bd17800000)]
  0x000002a7fbcae710 JavaThread "Cache worker for dependencies-accessors (D:\Anki-Android\.gradle\8.1.1\dependencies-accessors)" [_thread_blocked, id=51624, stack(0x000000bd17800000,0x000000bd17900000)]
  0x000002a7fbcaebe0 JavaThread "jar transforms Thread 5" [_thread_blocked, id=51540, stack(0x000000bd17900000,0x000000bd17a00000)]
  0x000002a7fbcaa880 JavaThread "Cache worker for Build Output Cleanup Cache (D:\Anki-Android\.gradle\buildOutputCleanup)" [_thread_blocked, id=44696, stack(0x000000bd17b00000,0x000000bd17c00000)]
  0x000002a7fbcaca30 JavaThread "Unconstrained build operations" [_thread_blocked, id=32828, stack(0x000000bd17c00000,0x000000bd17d00000)]
  0x000002a7fbcb03f0 JavaThread "Unconstrained build operations Thread 2" [_thread_blocked, id=34316, stack(0x000000bd17d00000,0x000000bd17e00000)]
  0x000002a7fbcb08c0 JavaThread "Unconstrained build operations Thread 3" [_thread_blocked, id=7064, stack(0x000000bd17e00000,0x000000bd17f00000)]
  0x000002a7fbcadd70 JavaThread "Unconstrained build operations Thread 4" [_thread_blocked, id=31252, stack(0x000000bd17a00000,0x000000bd17b00000)]
  0x000002a7fd2dff90 JavaThread "Unconstrained build operations Thread 5" [_thread_blocked, id=52332, stack(0x000000bd18000000,0x000000bd18100000)]
  0x000002a7fd2da8f0 JavaThread "Unconstrained build operations Thread 6" [_thread_blocked, id=20800, stack(0x000000bd18100000,0x000000bd18200000)]
  0x000002a7fd2de780 JavaThread "Unconstrained build operations Thread 7" [_thread_blocked, id=5084, stack(0x000000bd18200000,0x000000bd18300000)]
  0x000002a7fd2e0930 JavaThread "Unconstrained build operations Thread 8" [_thread_blocked, id=25836, stack(0x000000bd18300000,0x000000bd18400000)]
  0x000002a7fd2e0460 JavaThread "Unconstrained build operations Thread 9" [_thread_blocked, id=15940, stack(0x000000bd18400000,0x000000bd18500000)]
  0x000002a7fd2d90e0 JavaThread "Unconstrained build operations Thread 10" [_thread_blocked, id=11724, stack(0x000000bd18500000,0x000000bd18600000)]
  0x000002a7fd2df120 JavaThread "Unconstrained build operations Thread 11" [_thread_blocked, id=1876, stack(0x000000bd18600000,0x000000bd18700000)]
  0x000002a7fd2dcf70 JavaThread "Unconstrained build operations Thread 12" [_thread_blocked, id=35144, stack(0x000000bd18700000,0x000000bd18800000)]
  0x000002a7fd2df5f0 JavaThread "Unconstrained build operations Thread 13" [_thread_blocked, id=50928, stack(0x000000bd18800000,0x000000bd18900000)]
  0x000002a7fd2db290 JavaThread "Unconstrained build operations Thread 14" [_thread_blocked, id=21144, stack(0x000000bd18900000,0x000000bd18a00000)]
  0x000002a7fd2dec50 JavaThread "Unconstrained build operations Thread 15" [_thread_blocked, id=49708, stack(0x000000bd18a00000,0x000000bd18b00000)]
  0x000002a7fd2d9a80 JavaThread "Unconstrained build operations Thread 16" [_thread_blocked, id=22580, stack(0x000000bd18b00000,0x000000bd18c00000)]
  0x000002a7fd2dd910 JavaThread "Unconstrained build operations Thread 17" [_thread_blocked, id=40652, stack(0x000000bd18c00000,0x000000bd18d00000)]
  0x000002a7fd2dfac0 JavaThread "Unconstrained build operations Thread 18" [_thread_blocked, id=52392, stack(0x000000bd18d00000,0x000000bd18e00000)]
  0x000002a7fd2ddde0 JavaThread "Unconstrained build operations Thread 19" [_thread_blocked, id=42348, stack(0x000000bd19000000,0x000000bd19100000)]
  0x000002a7fd2da420 JavaThread "Unconstrained build operations Thread 20" [_thread_blocked, id=1488, stack(0x000000bd19100000,0x000000bd19200000)]
  0x000002a7fd2dc100 JavaThread "Unconstrained build operations Thread 21" [_thread_blocked, id=11776, stack(0x000000bd19200000,0x000000bd19300000)]
  0x000002a7fd2dadc0 JavaThread "Unconstrained build operations Thread 22" [_thread_blocked, id=52636, stack(0x000000bd19300000,0x000000bd19400000)]
  0x000002a7fd2dcaa0 JavaThread "Unconstrained build operations Thread 23" [_thread_blocked, id=50888, stack(0x000000bd19400000,0x000000bd19500000)]
  0x000002a7fd2db760 JavaThread "Unconstrained build operations Thread 24" [_thread_blocked, id=48552, stack(0x000000bd19500000,0x000000bd19600000)]
  0x000002a7fd2dbc30 JavaThread "Unconstrained build operations Thread 25" [_thread_blocked, id=41632, stack(0x000000bd18e00000,0x000000bd18f00000)]
  0x000002a7fd2dc5d0 JavaThread "Unconstrained build operations Thread 26" [_thread_blocked, id=46672, stack(0x000000bd18f00000,0x000000bd19000000)]
  0x000002a7fd2de2b0 JavaThread "Unconstrained build operations Thread 27" [_thread_blocked, id=41076, stack(0x000000bd19600000,0x000000bd19700000)]
  0x000002a7fba91790 JavaThread "Unconstrained build operations Thread 28" [_thread_blocked, id=46696, stack(0x000000bd19700000,0x000000bd19800000)]
  0x000002a7fba947b0 JavaThread "Unconstrained build operations Thread 29" [_thread_blocked, id=41476, stack(0x000000bd19800000,0x000000bd19900000)]
  0x000002a7fba91c60 JavaThread "Unconstrained build operations Thread 30" [_thread_blocked, id=40992, stack(0x000000bd19900000,0x000000bd19a00000)]
  0x000002a7fba93940 JavaThread "Unconstrained build operations Thread 31" [_thread_blocked, id=38236, stack(0x000000bd19a00000,0x000000bd19b00000)]
  0x000002a7fba94c80 JavaThread "Unconstrained build operations Thread 32" [_thread_blocked, id=48208, stack(0x000000bd19b00000,0x000000bd19c00000)]
  0x000002a7fba92130 JavaThread "Unconstrained build operations Thread 33" [_thread_blocked, id=52372, stack(0x000000bd19c00000,0x000000bd19d00000)]
  0x000002a7fba92ad0 JavaThread "Unconstrained build operations Thread 34" [_thread_blocked, id=47476, stack(0x000000bd19d00000,0x000000bd19e00000)]
  0x000002a7fba912c0 JavaThread "Unconstrained build operations Thread 35" [_thread_blocked, id=16152, stack(0x000000bd19e00000,0x000000bd19f00000)]
  0x000002a7fba92600 JavaThread "Unconstrained build operations Thread 36" [_thread_blocked, id=46336, stack(0x000000bd19f00000,0x000000bd1a000000)]

```

**Comment by criticalAY** [comment]
Screenshot_116
This is common when I run out of memory ( I left android studio running for days in the background while I was not working on AnkiDroid, left it open *exams lol* )

**Comment by criticalAY** [comment]
I am moving this out of draft for review

File: .gitignore [hunk 1]
```diff
@@ -58,3 +58,4 @@ tools/crowdin_key.txt
 # Testing / quality remnants
 AnkiDroid/ACRA-INSTALLATION
 .codacy.json
+*.log
```
Comment (lukstbit) [inline .gitignore:61]: This is an edge case but I'm ok with adding an entry in gitignore. However `*.log` is too general and I would ask to use a more specific ignore rule: `hs_err_pid*`.

**Review by lukstbit — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (criticalAY) [reply .gitignore]: umm ok it is more specific

**Review by lukstbit — APPROVED** [review]

**Review by BrayanDSO — APPROVED** [review]

**Pull request merged**